kotlinx-coroutines-core/common/src/flow/operators/Errors.kt

```diff
@@ -60,35 +60,6 @@ public fun <T> Flow<T>.catch(action: suspend FlowCollector<T>.(cause: Throwable)
         if (exception != null) action(exception)
     }
 
-/**
- * @suppress **Deprecated**: Use `(Throwable) -> Boolean` functional type
- */
-@Deprecated(
-    level = DeprecationLevel.ERROR,
-    message = "Use (Throwable) -> Boolean functional type",
-    replaceWith = ReplaceWith("(Throwable) -> Boolean")
-)
-public typealias ExceptionPredicate = (Throwable) -> Boolean
```

Comment:
Do I understand correctly that type aliases don't leave a trace in target binaries, so it doesn't make sense to make them hidden, as they would simply become useless?